components/dashboard/src/service/public-api.ts

```diff
@@ -16,7 +16,7 @@ import { TeamMember, TeamRole } from "@gitpod/public-api/lib/gitpod/experimental
 import { Project } from "@gitpod/public-api/lib/gitpod/experimental/v1/projects_pb";
 
 const transport = createConnectTransport({
-    baseUrl: `${window.location.protocol}//api.${window.location.host}`,
+    baseUrl: `${window.location.protocol}//${window.location.host}/public-api`,
     credentials: "include",
```

Comment:
I think this should no longer be needed, as it's the same origin so the cookies should be automatically included.